keanu-project/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,10 @@
+# Root logger option
```

Comment:
Just curious, is this related to this PR?

Reply:
it's related to me trying to debug why it was freezing on Windows!

Reply:
and it will be useful to have this in our main branch